flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -735,6 +735,14 @@ public Table sqlQuery(String query) {
     public TableResult executeSql(String statement) {
         List<Operation> operations = getParser().parse(statement);
 
+        if (operations.size() > 1
+                && operations.stream().allMatch(op -> op instanceof ModifyOperation)) {
+            // allow multi modification operation
+            return executeInternal(
+                    operations.stream()
+                            .map(op -> (ModifyOperation) op)
+                            .collect(Collectors.toList()));
+        }
         if (operations.size() != 1) {
             throw new TableException(UNSUPPORTED_QUERY_IN_EXECUTE_SQL_MSG);
```

Comment:
The message of this exception doesn't make sense if `operation.size() > 1` but not all operations are `ModifyOperation`. Can you improve the error message in such case?